Traffic light.py

```diff
@@ -122,7 +122,7 @@
 
 ##Task 4 
 
-
+# yes
```

Comment:
Honestly this looks p good. Other than saying you could probably have a file per task your doing good :).